.github/labels.yml

```diff
@@ -0,0 +1,25 @@
+dependencies:
+- flask/requirements*.txt
+- flask/requirements*.in
+- react/yarn.lock
+database:
+- flask/app/manage.py
+- flask/app/models.py
+- flask/migrations/**/*
+backend:
+- any: ["flask/**/*"]
+  all: ["!flask/app/manage.py", "!flask/app/models.py", "!flask/migrations/**/*"]
```

Comment:
If flask/app/manage.py and flask/something are changed, I don't think this will apply the backend label. Something like 
```
# Add 'source' label to any change to src files within the source dir EXCEPT for the docs sub-folder
source:
- any: ['src/**/*', '!src/docs/*']
```
might be more applicable

Reply:
Good catch! Should be fixed